### ISSUE TITLE
Chokes on yield from

### ISSUE THREAD

**Issue by jennolsen84** [google/yapf#132]
yapf chokes on the following python3 code:

```
import asyncio

@asyncio.coroutine
def x():
    yield from asyncio.sleep(1)
```

**Comment by hayd** [comment]
This seems to work for me, but **only if** you are installed/running yapf in python3. I think this is expected.

**Comment by eliben** [comment]
When running yapf on Python 3 code, yapf itself should be run under Python 3 (otherwise lib2to3 on which yapf relies is not guaranteed to support all of Python 3). See https://github.com/google/yapf#python-versions

**Comment by jennolsen84** [comment]
I am definitely running under python3.  Installed and ran from a venv.  Here is the stack trace:

```
Traceback (most recent call last):
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/yapflib/pytree_utils.py", line 100, in ParseCodeToTree
    tree = parser_driver.parse_string(code, debug=False)
  File "/usr/lib/python3.4/lib2to3/pgen2/driver.py", line 106, in parse_string
    return self.parse_tokens(tokens, debug)
  File "/usr/lib/python3.4/lib2to3/pgen2/driver.py", line 71, in parse_tokens
    if p.addtoken(type, value, (prefix, start)):
  File "/usr/lib/python3.4/lib2to3/pgen2/parse.py", line 159, in addtoken
    raise ParseError("bad input", type, value, context)
lib2to3.pgen2.parse.ParseError: bad input: type=1, value='from', context=(' ', (5, 10))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/vagrant/myvenv/bin/yapf", line 11, in <module>
    sys.exit(run_main())
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/__init__.py", line 209, in run_main
    sys.exit(main(sys.argv))
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/__init__.py", line 146, in main
    verify=args.verify)
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/__init__.py", line 174, in FormatFiles
    print_diff=print_diff, verify=verify)
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/yapflib/yapf_api.py", line 78, in FormatFile
    verify=verify), encoding
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/yapflib/yapf_api.py", line 103, in FormatCode
    tree = pytree_utils.ParseCodeToTree(unformatted_source)
  File "/home/vagrant/myvenv/lib/python3.4/site-packages/yapf/yapflib/pytree_utils.py", line 106, in ParseCodeToTree
    tree = parser_driver.parse_string(code, debug=False)
  File "/usr/lib/python3.4/lib2to3/pgen2/driver.py", line 106, in parse_string
    return self.parse_tokens(tokens, debug)
  File "/usr/lib/python3.4/lib2to3/pgen2/driver.py", line 71, in parse_tokens
    if p.addtoken(type, value, (prefix, start)):
  File "/usr/lib/python3.4/lib2to3/pgen2/parse.py", line 159, in addtoken
    raise ParseError("bad input", type, value, context)
lib2to3.pgen2.parse.ParseError: bad input: type=1, value='from', context=(' ', (5, 10))
```

**Comment by eliben** [comment]
What is the exact version of your Python? I think this was fixed in 3.4.1, which is why this is the minimal required version stated in the README of yapf.

**Comment by jennolsen84** [comment]
That was it.  Thank you
